### ISSUE TITLE
Having trouble with directories, any advice please?

### ISSUE THREAD

**Issue by ThomasFan1945** [haoheliu/versatile_audio_super_resolution#11]
![image](https://github.com/haoheliu/versatile_audio_super_resolution/assets/66193286/98e513c4-e068-472a-a7b8-6d988461fc9e)

**Comment by madwurmz** [comment]
yes, I have this trouble too , confusing! I don't understand why it will install in  .\anaconda3\envs\ 
shouldn't it install like any other repo, in any given and dedicated directory? 
I wonder if this install process is some big mistake?
